packages/@adobe/react-spectrum-uxp/README.md

```diff
@@ -0,0 +1,8 @@
+# [React Spectrum UXP](https://react-spectrum.adobe.com/react-spectrum/index.html)
+
+A React implementation of Spectrum, Adobe’s design system.
+
+This version provides an API compatible set of components designed to run on Adobe’s Unified Extensibility Platform.
+
+See the main React Spectrum docs for more information.
```

Comment:
I've added a link to the external UXP documentation. I'm not sure what to add regarding setting up this repo because it compiles and is used just like every other repo within this monorepo. This specific repo is homologous with @adobe/react-spectrum. I'll add more specific setup documentation in the @react-spectrum-uxp/storybook repository.